sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -26,7 +27,8 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
   constructor(tokenCredentialOptions?: TokenCredentialOptions) {
     super(
       new EnvironmentCredential(tokenCredentialOptions),
-      new ManagedIdentityCredential(tokenCredentialOptions)
+      new ManagedIdentityCredential(tokenCredentialOptions),
+      new AzureCliCredential()
```

Comment:
PR desc says it is not included in DAC.  But this code looks like it is.  

It is included with Python.  Please sync with rest of lang devs to ensure we are on the same page across langs

I highly prefer it to be enabled by default so developer doesn't have to do anything to configure it.

Reply:
Looping @schaabs to confirm we want it to be part of the DAC. Last we talked, he was looking into a set of credentials that are part of the Developer Credentials, but it didn't sound like it was expected to be part of the default just yet.

Scott - can you confirm this should be part of the DAC?

Reply:
Just chatted with Scott, it *should* be part of the DAC so I'll just revise the commit message and leave the code as is.  Sorry for the confusion.